rust-runtime/aws-smithy-http-server/src/macros.rs

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! Macros implementation
+
+// This file is a copy of Axum's  macros.rs (https://github.com/tokio-rs/axum/blob/2507463706d0cea90007b5959c579a32d4b24cc4/axum/src/macros.rs#L1)
+// with simple modification to allow public visibility and re-use of the generated structures.
+// Axum original license can be found inside the file named LICENSE.mit.
```

Comment:
see `aws-smithy-async/future/timeout.rs` for what was the official way to attribute

Reply:
Ah, did not know, thanks I'll look into it.

Reply:
Fixed!